src/Symfony/Bundle/FrameworkBundle/FrameworkBundle.php

```diff
@@ -97,6 +98,15 @@ public function boot()
         }
     }
 
+    public function shutdown()
```

Comment:
This fixes the issue but I don't expect this is the proper solution. I'd like some input on where this fix should be made.